### PR TITLE
fix: incorrect description in WebExtensions

### PR DESCRIPTION
**Pull request by PassionPenguin**
```markdown
-  - : `object`. An object containing either `imageData` or `path` properties, and optionally a `tabId` property.
+  - : `object`. An object containing either `imageData` or `path` properties, and optionally `tabId` and/or `windowId` properties.
```

### CODE REVIEW

**Review by Rob--W — APPROVED** [review]
The changes looks good to me from the technical perspective.

@rebloor Does the language look good to you too?

File: files/en-us/mozilla/add-ons/webextensions/api/action/seticon/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ let settingIcon = browser.action.setIcon(
 
 - `details`
 
-  - : `object`. An object containing either `imageData` or `path` properties, and optionally `tabId` and/or `windowId` propertyies.
+  - : `object`. An object containing either `imageData` or `path` properties, and optionally `tabId` and/or `windowId` properties.
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/action/seticon/index.md:35]:
```suggestion
  - : `object`. An object containing the `imageData` or `path` property and, optionally, either or both of the `tabId` and `windowId` properties.
```

File: files/en-us/mozilla/add-ons/webextensions/api/browseraction/seticon/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ let settingIcon = browser.browserAction.setIcon(
 
 - `details`
 
-  - : `object`. An object containing either `imageData` or `path` properties, and optionally a `tabId` property.
+  - : `object`. An object containing either `imageData` or `path` properties, and optionally `tabId` and/or `windowId` properties.
```
Comment (PassionPenguin) [inline files/en-us/mozilla/add-ons/webextensions/api/browseraction/seticon/index.md:32]:
```suggestion
  - : `object`. An object containing the `imageData` or `path` property and, optionally, either or both of the `tabId` and `windowId` properties.
```

**Comment by rebloor** [comment]
Thanks @PassionPenguin

**Pull request merged**